apps/api/test/functional/deployments.spec.ts

```diff
@@ -145,19 +127,67 @@ describe("Deployments API", () => {
     return { adminApiKeySecret };
   }
 
-  afterEach(async () => {
-    await dbService.cleanAll();
-  });
+  function createMockDeploymentInfo(wallets: UserWalletOutput[], dseq: string, deploymentInfo?: RestAkashDeploymentInfoResponse) {
```

Comment:
non-blocking: it would be nice if extracted to a seeder

Reply:
non-blocking: also faker is a good thing where it makes sense

Reply:
I will keep working on those tests in my current create lease branch 👍